### DISCUSSION TITLE
WIP Categories spreadsheet and mapping to cloud providers offering

### DISCUSSION THREAD

**Discussion by vicenteherrera** [finos/common-cloud-controls#29] [Show and tell]
Hi friends,

I want to start this discussion thread sharing an updated version of [the spreadsheet I started](https://github.com/finos/common-cloud-controls/issues/15#issuecomment-1708304482), with categories originally proposed by @simonzhangbmo [here](https://github.com/finos/common-cloud-controls/issues/15#issuecomment-1701530139), and my first attempt to map them to what cloud providers use for categories.

This is an improved version where I've found a better source for official AWS categories, they map a lot better (almost perfeclty) than my previous attempt.
https://docs.google.com/spreadsheets/d/10TP21qAnjnn5TkW1y758oPpjzc4CcyF4UKNCmsnlzps/edit#gid=1772808763

I also attacht here an Excel file for it.
[Finos CCC categories - 2023-09-13.xlsx](https://github.com/finos/common-cloud-controls/files/12602104/Finos.CCC.categories.-.2023-09-13.xlsx)

I know this has to be done under consensus, my intention with this is having something "shitty" to start with that we can iterate and see how it fits. We should stablish a good criteria about the conditions for separation of categories we are defining (why is containers not compute), and agree to a couples of paragraphs description for each one. We should also list in more detail each service inside each cloud provider category, and make sure not only names match but also the category definition.

But until we solve how to organize work, allow me to tinker with this and share it with you here. It's completely open to anybody's feedback and suggestions... let's discuss, friends!

**Comment by mcleo-d** [comment]
@vicenteherrera - I have added this discussion to the CCC project kanban - https://github.com/orgs/finos/projects/78/views/1

Thanks for raising 🙌🏻 

cc @rhyddian and @mark-rushing

**Reply by smendis-scottlogic** [reply]
Hi @vicenteherrera, 

Me and my colleagues, @eziogas-scottlogic and Robert Griffiths took the document you shared for categories and explored storage options across all CSPs. This is still WIP. But thought of sharing it to check what you think about it.

[Comparison - AWS, Azure, GCP.xlsx](https://github.com/finos/common-cloud-controls/files/12617947/Comparison.-.AWS.Azure.GCP.xlsx)

Thank you

cc @rhyddian and @mark-rushing

**Reply by mcleo-d** [reply]
Hi @smendis-scottlogic - The link above is asking for a Scott Logic login. Also, I've noticed there is an emoji rendered in the URL :)

**Reply by smendis-scottlogic** [reply]
Hi @mcleo-d sorry about that. will remove that link. Can you download the attachment?

**Reply by mcleo-d** [reply]
Hi @smendis-scottlogic

I have just raised the following pull request that converts @vicenteherrera's original work to markdown.

https://github.com/finos/common-cloud-controls/pull/39

Is there a big difference between your Excel and the original sheet as like to make sure we don't lose track of versions?

Many thanks,

James.

**Reply by mcleo-d** [reply]
Hi @smendis-scottlogic - I have taken your Excel sheet and have added to @vicenteherrera's Google Sheet below to ensure that your work doesn't get lost in Discussion comments...

https://docs.google.com/spreadsheets/d/10TP21qAnjnn5TkW1y758oPpjzc4CcyF4UKNCmsnlzps/edit?pli=1#gid=1458499499&range=A1

**Reply by mcleo-d** [reply]
Hi @smendis-scottlogic - I have added your CSP comparison to pull request #39 so it appears in the repo as markdown for people to extend and comment upon.

https://github.com/finos/common-cloud-controls/pull/39/commits/77ed1fb6afd43f3b5080637f04a1c3857ff90bbd#r1337550928

**Comment by smendis-scottlogic** [comment]
Hi James,

Thanks! Hopefully we will be able to see whether we are on the right track after tomorrow's meeting.

Best Regards,
Sonali Mendis
________________________________

**Comment by damienjburks** [comment]
@smendis-scottlogic are we good to close this out?
